src/main/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiService.scala

```diff
@@ -155,6 +155,13 @@ trait AdminApiService extends HttpService with PerRequestCreator with UserInfoDi
           WorkspaceService.props(workspaceServiceConstructor, userInfo),
           WorkspaceService.RevokeAllUserReadAccess(WorkspaceName(workspaceNamespace, workspaceName)))
       }
+    } ~
+    path("admin" / "validate" / Segment / Segment / Segment) { (workspaceNamespace, workspaceName, userSubjectId) =>
```

Comment:
maybe put the user subject id in the query parameters, especially if it is optional